server/routerlicious/packages/services-utils/src/scheduleJob.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+import * as cron from "node-cron";
+import { ILogger } from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
+
+export async function scheduleJob<T>(
+    api: () => Promise<T>,
+    config: Provider,
+    featureGateConfigName: string,
+    callName: string,
+    cronExpressionConfigName: string,
+    logger?: ILogger,
```

Comment:
We can probably remove `ILogger` since from now onwards we will be using `Lumberjack`

Reply:
Oh okay sounds good